test-result-summary-client/src/Build/TopLevelBuilds.jsx

```diff
@@ -63,25 +63,18 @@ export default class TopLevelBuilds extends Component {
 
             const renderBuild = ( value, row, index ) => {
                 if ( value && value.buildNum ) {
-                    let result = value.buildResult;
-                    if ( value.tests && value.tests.length > 0 ) {
-                        result = value.tests[0].testResult;
-                        if ( value.tests[0]._id ) {
-                            return <div>
-                                <Link to={{ pathname: '/output/test', search: params( { id: value.tests[0]._id } ) }}
-                                    style={{ color: result === "PASSED" ? "#2cbe4e" : ( result === "FAILED" ? "#f50" : "#DAA520" ) }}>
-                                    Build #{value.buildNum}
-                                </Link>
-                            </div>;
-                        }
-                    } else {
-                        return <div>
-                            <Link to={{ pathname: '/output/all', search: params( { id: value._id } ) }}
-                                style={{ color: result === "SUCCESS" ? "#2cbe4e" : ( result === "FAILURE" ? "#f50" : "#DAA520" ) }}>
-                                Build #{value.buildNum}
-                            </Link>
-                        </div>;
-                    }
+                    return <div>
+                        <Link to={{ pathname: '/buildDetail', search: params( { parentId: value._id } ) }}

```

Comment:
We need to make sure the updated code will work for other types of tests.

Reply:
Modified as required. Please check.